### ISSUE TITLE
Website fails to fully load properly since 1.0.0

### ISSUE THREAD

**Issue by benoit74** [webrecorder/browsertrix-crawler#522]
I'm doing some tests on a demo Confluence website: https://octo-help.refined.com (not sure that the fact it is a demo Confluence has some importance).

The website seems to fail to load properly since 1.0.0 release.

This command succeeds to extract multiple pages:
```
docker run --rm webrecorder/browsertrix-crawler:0.12.3 crawl --url "https://octo-help.refined.com"
```

This commands loads only seed page and then stops, obviously failing to wait for page to be fully loaded
```
docker run --rm webrecorder/browsertrix-crawler:1.0.0 crawl --url "https://octo-help.refined.com"
```

I experimented with `--waitUntil load,networkidle0` without any success.

I experimented with various 1.0.0 beta images and 1.1.0 beta images, without any success as well.

I created a login profile with 1.1.0-beta.3 and the website is fully loaded in the interactive VNC display, so it is clearly working. Passing the login profile to the crawl does not help.

**Comment by tw4l** [comment]
Hi @benoit74 , my guess is that you hit the same issue as described in https://github.com/webrecorder/browsertrix-crawler/issues/519. Basically, the new version of the crawler loads the page so quickly that not all pages have time to sufficient time to render their content.

The just-released [1.1.0-beta.3 release](https://github.com/webrecorder/browsertrix-crawler/releases/tag/v1.1.0-beta.3) includes a new `--postLoadDelay` flag which can be used to pause the crawler x seconds once the page resources load before moving on to link extraction, screenshots, etc. - I think this should solve your problem if you want to try it out and report back.

**Comment by ikreymer** [comment]
Thanks for the report, this should now be fixed in v1.0.4 release!

**Comment by benoit74** [comment]
I confirm that v1.0.4 is now working, thank you very much!
